addons/product/tests/test_variants.py

```diff
@@ -401,6 +401,26 @@ def test_open_product_form_with_default_uom_id_is_false(self):
         product = product_form.save()
         self.assertEqual(uom_unit, product.uom_id)
 
+    def test_create_product_with_packaging_one_variant(self):
+        """Test what happen when creating a product with attribute,
+        with only one variant and a packaging. On creating the packaging
+        should not be duplicated."""
+
+        product_template = self.env['product.template'].create({
+            'name': 'one variant template',
+            'attribute_line_ids': [
+                Command.create({
+                    'attribute_id': self.size_attribute.id,
+                    'value_ids': [Command.set(self.size_attribute_s.ids)],
+                })],
+            'packaging_ids': [
+                Command.create({
+                    'name': 'packaging',
+                    }),
+            ],
+        })
+        self.assertEqual(len(product_template.product_variant_id.packaging_ids), 1)
```

Comment:
```suggestion
        self.assertEqual(len(product_template.packaging_ids), 1)
```
May as well ensure that the `packaging_ids` field also gets properly set on the product template.